fixed broken boilerplate

rlert opened the pull request:
body and footer elements now show properly

rlert committed:
fixed broken boilerplate

body and footer elements now show properly

hthm00 commented on views/partials/body.ejs:
this file cannot reuse so it should not be in partials, move the contents to pages>index.ejs

hthm00 reviewed (commented):
see below

rlert committed:
moved files from \partial to \pages

moved files from \partial to \pages, and changed \pages\index.ejs directory to new location

rlert committed:
Revert "fixed broken boilerplate"

This reverts commit 3a3a5896c92d8c5940f803f64f55aee70ca0fdb2.

rlert committed:
revert changes and created moved body

reverted changes to previous build and moved body.ejs contents to index.ejs

hthm00 reviewed (commented):
i don't see footer on home page

hthm00 reviewed (approved):
It's okay, Tony will update the footer later. Good work overall!
